static/styles/zulip.css

```diff
@@ -1302,6 +1302,13 @@ td.pointer {
     color: hsl(200, 100%, 40%);
 }
 
+.actions_hover {
+    i {
+        /* Additional padding to increase click-able area */
+        padding: 2px 4px;
+    }
+}
```

Comment:
Can you put this in a separate commit?  In any case I don't think it works as intended (see the note above).

Reply:
Ok , increased the padding a bit more. Should be fine now.